sunpy/database/attrs.py

```diff
@@ -85,14 +85,12 @@ def __repr__(self):
 
 
 class Path(Attr):
-    def __init__(self, value):
+    def __init__(self, value, inverted=False):
         self.value = value
-        self.inverted = False
+        self.inverted = inverted
 
     def __invert__(self):
-        path = self.__class__(self.value)
-        path.inverted = True
-        return path
```

Comment:
I preserved the same behavior in the changes, but in my opinion the `__inverted__` should be bijective, such as `-(-(1)) = 1`

Reply:
What do you think @nabobalis?

Reply:
I think it would be better to leave its current behaviour for now and open an issue about if we want to change this down the road.